trs_filer/ga4gh/trs/endpoints/registerObjects.py

```diff
@@ -0,0 +1,73 @@
+"""Controller for adding new objects."""
+
+from random import choice
+from typing import Dict
+from trs_filer.app import logger
+from flask import (current_app, request)
+from pymongo.errors import DuplicateKeyError
+
+
+class CreateToolPostObject:
+    def __init__(self, request: request):
```

Comment:
Please explicitly mention that you return `None` type. Always type the returns, as well as all params.